### PR TITLE
generate_figure.py: make "results" an optional input param; other improvements

### LINKED ISSUE #111: Update doc with new behaviour without paramers.sh
here: https://spine-generic.readthedocs.io/en/latest/documentation.html#generate-figures

the doc instructs to use a parameters.sh file, but we don't have that anymore. So, doc should be updated

### PR DESCRIPTION
**Pull request by valosekj**
This PR introduces the flag `-path-results` to the csv folder, for more convenient use. Fixes: #63 

Additional improvements include:
- Clarification of computation for intra/inter-COV
- Update documentation to reflect the drop of the parameters.sh file. Fixes #111

### CODE REVIEW

File: processing/generate_figure.py [hunk 1]
```diff
@@ -199,6 +199,11 @@ def get_parameters():
         required=False,
         help="Display the value of each individual subject as a red dot.",
         default=1)
+    parser.add_argument(
+        '-results',
```
Comment (jcohenadad) [inline processing/generate_figure.py:203]: i would call it "-path-results"  to be explicit

**Comment by jcohenadad** [comment]
@valosekj is this ready to be reviewed?

**Comment by valosekj** [comment]
> @valosekj is this ready to be reviewed?

@jcohenadad Yes, it is. I have tested it again just now and it works fine.

**Comment by valosekj** [comment]
> @valosekj as you can see i removed the duplicated usage description between the function header and the usage-- hope that's ok with you

Yes, it is ok. Thank you 👍

**Pull request merged**